integration-cli/docker_cli_run_test.go

```diff
@@ -2516,3 +2516,27 @@ func (s *DockerSuite) TestRunWriteFilteredProc(c *check.C) {
 		}
 	}
 }
+
+func (s *DockerSuite) TestRunNetworkFilesBindMount(c *check.C) {
+	testRequires(c, SameHostDaemon)
+	name := "test-nwfiles-mount"
+
+	f, err := ioutil.TempFile("", name)
+	c.Assert(err, check.IsNil)
+
+	filename := f.Name()
+	defer os.Remove(filename)
+
+	expected := "test123"
+
+	err = ioutil.WriteFile(filename, []byte(expected), 0644)
+	c.Assert(err, check.IsNil)
+
+	var actual string
+	actual, _, err = dockerCmdWithError(c, "run", "-v", filename+":/etc/resolv.conf", "busybox", "cat", "/etc/resolv.conf")
```

Comment:
Sorry I'm being a stickler here, but there is no reason to use `dockerCmdWithError` here.
`dockerCmd` checks the error and fails the test automatically.
So unless the error needs to be inspected you should use `dockerCmd`.